### PR TITLE
🐛Remove start.sh and restart.sh

### LINKED ISSUE #760: make image stuck time to time when download start.sh and restart.sh
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

build stuck here time to time though I can access internet..
I searched and didn't found start.sh and restart.sh usage .. so can we remove them? 
```
---> cb7ce8bbd8c7
Step 8/18 : COPY ./ ./
 ---> 46cd4b39728b
Step 9/18 : RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh  &&     wget --output-document /start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh &&     chmod +x /start.sh && chmod +x /restart.sh
 ---> Running in f65270b870e9
^CMakefile:205: recipe for target 'docker-build' failed
make: *** [docker-build] Interrupt

```

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster API Provider OpenStack version (Or `git rev-parse HEAD` if manually built):
- Cluster-API version:
- OpenStack version:
- Minikube/KIND version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #760

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

/hold

### CODE REVIEW

File: Dockerfile
Comment (sbueringer) [inline Dockerfile:32]: Why do you want to delete this?

**Review by sbueringer — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (sbueringer) [reply Dockerfile:32]: Okay read your comment. Some important notes:
* this is not the final image which we realease
* it's absolutelty essential for development: https://cluster-api.sigs.k8s.io/developer/tilt.html

Everyone else has it. I would really like to keep it

Comment (sbueringer) [inline Dockerfile:32]: Ups deleted my own comment :/

We need this for development: https://cluster-api.sigs.k8s.io/developer/tilt.html
Tilt uses it to live-reload the binary when rebuilding.

**Review by sbueringer — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (sbueringer) [reply Dockerfile:32]: Ah maybe it's not necessary anymore because it's added by tilt. I have to test it:
https://github.com/kubernetes-sigs/cluster-api/blob/master/Tiltfile#L127-L142

Comment (sbueringer) [reply Dockerfile:32]: Okay tested it. Seems like with v1alpha4 they solved this by using a separate dockerfile generated in the Tiltfile.

So while this is a breaking change for development with Tilt on master. It's fine for me as it won't be necessary after we hopefully merged the v1alpha4 PR soon. I also synced the Dockerfile with the upstream cluster-api Dockerfile on the v1alpha4 PR. There seem to be a lot more performance improvements there.: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/748/commits/2fc696d2f610dc920a08f5844b5aad8bbae38a20

**Review by sbueringer — APPROVED** [review]

File: Dockerfile
Comment (jichenjc) [reply Dockerfile:32]: ok, I don't know it's from Tilt :)

thanks and it brings trouble to me (can't download those files and have to manually update dockerfile when build image)

**Comment by sbueringer** [comment]
@jichenjc In my opinion you can merge it if you want (doesnt' conflict with my PR, in fact mine also does it)

**Comment by jichenjc** [comment]
@sbueringer please lgtm to this PR ..

/hold cancel

**Pull request merged**
